spec/models/user_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+
+
+  context 'default user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.